docs/src/tutorials/bseries_basics.md

```diff
@@ -0,0 +1,314 @@
+# [B-Series basics](@id tutorial-bseries-basics)
+In this tutorial we use `bseries.jl` to investigate error expansions for RK
+methods, generically or when applied to a specific ODE.
+
+
+```julia
+# Load the packages we will use.  These must first be installed using: import Pkg; Pkg.add("package_name")
+using BSeries
+using Latexify  # Only needed for some pretty-printing cells below
+import SymPy; sp=SymPy;
+```
+
+# B-Series for a generic ODE
+
+First we specify the Butcher coefficients of the RK method.
+This can include symbolic expressions and parameterized families of methods.
+Here is a generic 2-stage, 2nd-order method:
+
+
+```julia
+α = sp.symbols("α", real=true)
+A = [0 0; 1/(2*α) 0]; b = [1-α, α]; c = [0, 1/(2*α)]
+coeffs2 = bseries(A,b,c,3)
+latexify(coeffs2, cdot=false)
+```
+
+
+
+
+$F_{f}\mathopen{}\left( \varnothing \right)\mathclose{} + h F_{f}\mathopen{}\left( \rootedtree[] \right)\mathclose{} + \frac{h^{2}}{2} F_{f}\mathopen{}\left( \rootedtree[[]] \right)\mathclose{} + \frac{h^{3}}{8 \alpha} F_{f}\mathopen{}\left( \rootedtree[[][]] \right)\mathclose{}$
+
+
```

Comment:
````suggestion
```@example ex:bseries-basics
α = sp.symbols("α", real=true)
A = [0 0; 1/(2*α) 0]; b = [1-α, α]; c = [0, 1/(2*α)]
coeffs2 = bseries(A,b,c,3)
latexify(coeffs2, cdot=false)
```
````
This is how it would look like as an `@example` block. Documenter.jl will then run this code, continuing the `using` statements from above and should render the output of the last expression for our docs.